### DISCUSSION TITLE
Durata batteria UPS

### DISCUSSION THREAD

**Discussion by avvalex** [indomus/forum#2917] [GENERALE - Discussioni generiche]
Buongiorno ragazzi, qualcuno saprebbe indicarmi quale sia la durata media di una batteria per UPS, e se ci sono accorgimenti pet prolungarla?
Ho un classico Tecnoware 1100 che alimenta mini PC, modem e un altro paio di cosette, per un carico totale dell'UPS pari al 10-12%. Con batteria nuova resiste direi almeno mezz'ora, però tempo un anno o poco più e mi ritrovo a piedi, con la tensione che salta istantaneamente (batteria andata). 
Non sono pochi 12-15 mesi? Ho provato batterie di marche diverse, da 15 a 40 euro, ma senza sostanziali differenze.
L'UPS non è chiuso in qualche mobile, non interviene praticamente mai e non gli faccio cicli di scarica.... E' qui che sbaglio?
Fatemi sapere, grazie!

**Comment by KC-inDomus** [comment]
ciao, la domanda è troppo generica: la durata è in funzione della sua capacità e dal carico che gli applichi. leggiti questo: https://indomus.it/guide/come-scegliere-un-ups-per-la-propria-domotica-e-altro/
sul fatto dei clicli di carica e scarica e della durata nel tempo 🤷‍♂️ penso che molto dipenda dalla qualità e tipologia tecnologica della batteria. un anno mi pare poco, di tecnoware 1100 ne ho due e la prima batteria l'ho cambiata mi pare dopo due anni e mezzo circa (vado a memoria).

**Reply by 50494554524F** [reply]
è il tipo di batterie che dura poco, anche negli allarmi non durano... le migliori sono le yuasa, se la cambi prendi queste
